WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -974,6 +997,14 @@ - (void)loginValidationFailedWithError:(NSError *)error
     }
 }
 
+- (NSString *)getTagsCountPresentableString:(NSInteger)tagCount
+{
+    NSString *format = tagCount == 1 ? NSLocalizedString(@"%@ Tag", @"The number of tags in the writting settings. Singular.") : NSLocalizedString(@"%@ Tags", @"The number of tags in the writting settings. Plural.");
```

Comment:
Our translators might not understand the significance of the `%@`.  Maybe mention its a placeholder in the description.

I dig the user of the trinary operator, but the line is getting pretty long, so its not too readable at a glance.  Could we make a change to improve readability?

Reply:
I tried to reformat the ternary operator, but I was not convinced, so I did a traditional if/else. You can check the last two commits, see which version looks more readable to you.